packages/__docs__/src/CodeSandboxButton/index.tsx

```diff
@@ -73,14 +73,15 @@ import 'moment/min/locales'
       ', '
     )}} from "@instructure/ui"\n\n`
 
-    const codeLines = this.props.code.split('\n')
-    const shouldRender = !codeLines[codeLines.length - 1]
-      .split(' ')
-      .join('')
-      .includes('render(<')
-    const codeBlock = shouldRender
-      ? `render(${this.props.code})`
-      : this.props.code
+    const cleanedCode = this.props.code.replace(/\s+/g, '')
```

Comment:
regexp are so hard to read, please add here a comment on what this one does

Reply:
comment added